addons/mail/static/src/core/common/store_service.js

```diff
@@ -426,17 +432,24 @@ export class Store extends BaseStore {
                 });
             partner_ids.push(...recipientIds);
         }
+        const postData = {
+            body: await prettifyMessageContent(body, validMentions),
+            attachment_ids: attachments.map(({ id }) => id),
+            message_type: "comment",
+            partner_ids,
+            subtype_xmlid: subtype,
+        };
+        if (thread.model === "discuss.channel") {
+            postData.body = `<odoo-markdown>${postData.body}</odoo-markdown>`;
+            if (validMentions?.specialMentions) {
+                postData.special_mentions = validMentions.specialMentions;
```

Comment:
probably a relic of retargetting from master to 17.4 which now precedes this change: https://github.com/odoo/odoo/pull/173868